persistence-test/src/main/java/io/stargate/it/storage/IsDseCondition.java

```diff
@@ -0,0 +1,22 @@
+package io.stargate.it.storage;
+
+import com.datastax.oss.driver.api.testinfra.ccm.CcmBridge;
+import org.junit.jupiter.api.extension.ConditionEvaluationResult;
+import org.junit.jupiter.api.extension.ExecutionCondition;
+import org.junit.jupiter.api.extension.ExtensionContext;
+
+/**
+ * A condition that checks if the persistence backend uses DSE.
+ *
+ * <p>The test/suite is skipped if the persistence backend is not running DSE.
+ */
+public class IsDseCondition implements ExecutionCondition {
+  @Override
+  public ConditionEvaluationResult evaluateExecutionCondition(ExtensionContext extensionContext) {
+    if (CcmBridge.DSE_ENABLEMENT) {
```

Comment:
Could we get the current cluster (the same way `StargateContainer` gets it) and use `ClusterConnectionInfo.isDse()` instead? I think avoiding static references would be nice in this context.